tests/AuthFactoryTest.php

```diff
@@ -29,6 +29,10 @@ public function testNewAuthWithSessionAndSegment()
 
     public function testNewPdoAdapter_passwordVerifier()
     {
+        if (false === extension_loaded('pdo_sqlite')) {
```

Comment:
I like that idea, will send a new commit at lunchtime or after work.

Thanks for the fast reply!